typings/custom-global/index.d.ts

```diff
@@ -52,3 +52,10 @@ declare interface SetConstructor {
 }
 /** Partial Set constructor representing what's available in IE 11 */
 declare var Set: SetConstructor;
+
+/** webpack "mode MACROS" */
```

Comment:
Would you mind adding a comment above this explaining what it's doing (or a doc link)?

Reply:
Yup can and will do!

Reply:
Ready for re-review

Reply:
That helps, thanks! When I first read this, I thought for some reason that the comment itself was a magical webpack thing (since some things in webpack do use special comments/substrings). A bit silly in retrospect, but I suppose it proves the expanded comment's usefulness.